src/main/cdp/launcher.ts

```diff
@@ -27,13 +27,15 @@ export class ChromeLauncher {
             noDefaultArgs: false,
             terminateProcessOnExit: true,
             connectionTimeout: 5000,
+            chromeExtensions: [],
```

Comment:
Wondering if this should have a default with the name of the extension? Probably workers and drones could avoid setting this extra argument if they don't need to?

Reply:
TBH I find the implemented approach with relative paths to node_modules ugly, but the path is resolved in runtime and  `require.resolve('module');` doesn't work in esm so I just went with the simplest solution. The path to the node_modules is also different in drone and worker repo.